backend/typescript/rest/reviewRoutes.ts

```diff
@@ -80,18 +80,24 @@ reviewRouter.get(
   },
 );
 
-reviewRouter.delete(
-  "/:id",
-  isAuthorizedByRole(new Set(["Admin"])),
```

Comment:
why is this middleware being removed?

Reply:
I must've missed it in the merge conflict resolution. I'll them back